challenges/objects-arrays.js

```diff
@@ -48,22 +85,61 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
 
-Once you have the new array created, sort the universities alphabetically and log the result. */
+Once you have the new array created, sort the universities alphabetically and log the result. 
+
 const universities = [];
 console.log(universities)
+*/
+
+let unis = [];
+graduates.forEach(function (object) {
+    let school = object.university;
+    unis.push(school)
+})
+
+const universities = [];
+console.log(unis)
+
+for(let i = 0; i < graduates.length; i++){
+    unis.sort()
+
+}
+
+console.log(unis)
+
+
 
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
 
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
 
-Log the result of your new array. */
+Log the result of your new array. 
+
 const contactInfo = [];
 console.log(contactInfo);
+*/
 
 
-/* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
+const contactInfo = [];
+    graduates.forEach(function (object) {
+        let name = object.first_name + " " + object.email;
+        contactInfo.push(name)
+        
+    console.log(contactInfo)
```

Comment:
Small oops, your console.log is in the for loop, causing it to print every time it iterates